dataprofiler/data_readers/graph_data.py

```diff
@@ -37,17 +95,14 @@ def _find_target_string_in_column(self, column_names, keyword_list):
                         break
             if has_target:
                 break
-        
         return target_index
 
-
     @classmethod
     def csv_column_names(cls, file_path, options):
```

Comment:
right now this presumes the first row is the header. Some CSVs have comments at the top. We cause use the info from `CSVData.is_match` and options to determine this. (will also need a test, e.g. single graph data file which has comments before the column names start).
to make this more generic we could do:
```python
@classmethod
def csv_column_names(cls, file_path, delimiter, quotechar, header): 
    column_names = []
    if header is None:
        return column_names

    for _ in range(header + 1):
        row = next(csv_reader)
    .
    .
    .
```
csv.reader can be updated to handle quotechar too:
```python
csv_reader = csv.reader(csv_file, delimiter=delimiter, quotechar=quotechar)
```